Feature ts

ti-tanino opened the pull request:
変更点は２点
・飲み会作成画面から一覧画面へのモーダルを閉じるボタン追加
・飲み会TBLの作成と、一覧画面へテスト的な呼出コードを記載

Commit:
feature確認

Commit:
飲み会作成画面　閉じるボタン配置

Commit:
飲み会TBL作成　TBL読み込み時点で頓挫

nagasakiTechLab commented on DuesCalculateApp/Model/DB/PartyTable.swift:
```diff
@@ -0,0 +1,22 @@
+import RealmSwift
+
+final class PartyTable: Object {
+    dynamic var partyId = Int() //飲み会ID
```
Realmのオブジェクトは初期化せずに型宣言だけしとくのがよいですね。
`dynamic var partyId: Int`
みたいな形です。
初期値与えるなら
`dynamic var partyId: Int = 0`
とか。

nagasakiTechLab commented on DuesCalculateApp/View/PartyAdd/PartyAddViewController.swift:
```diff
@@ -15,8 +15,18 @@ class PartyAddViewController: UIViewController {
+        
     }
+    
+        func tapCloseButton() {
```
インデントずれとるよ

nagasakiTechLab commented on DuesCalculateApp/View/PartyList/PartyListViewController.swift:
```diff
@@ -45,15 +46,29 @@ class PartyListViewController: UIViewController, UITableViewDelegate, UITableVie
+        try! realm.write() {
+            realm.add(testParty)
+        }
+        let testTitle = realm.objects(PartyTable.self).first
```
`.self`必要だっけ？

ti-tanino replied:
警告が出てたので追加しました。

nagasakiTechLab commented on DuesCalculateApp/Model/DB/PartyTable.swift:
```diff
@@ -0,0 +1,22 @@
+import Foundation
+import RealmSwift
+
+final class PartyTable: Object {
```
Realmオブジェクトのクラス名にTableは不要かな

nagasakiTechLab reviewed (changes requested):
見ました。
たぶんPartyTableがPJに追加されてないのが主要因だと思いますが、コメントの点も修正した方がいいと思います。

Commit:
【修正点】
・飲み会オブジェクトの名称からTBLを削除
・オブジェクトの宣言形式を修正し、初期値を与える
・プロジェクトにParty.swiftを認識させる
・テスト呼び出しのオブジェクト宣言位置を変更

ti-tanino commented:
@nagasakiTechLab 修正してテストコードが稼働したのであげ直しました！

Commit:
テストコードを削除
飲み会作成画面の閉じるボタンと
Partyオブジェクトをマージしたいです。
